### PR TITLE
#16 activities list view -Victoria

### LINKED ISSUE #16: Create a list view of activities
As a user,
When I visit `/activities`
I see a list of activities and their info

- Activities list is hardcoded data on the frontend
- Activities info includes name, description, vote count, category
- No styling included

### PR DESCRIPTION
**Pull request by Victoria240**
Solves Issue #16

### CODE REVIEW

File: src/ActivityList.tsx
Comment (angelkbrown) [inline src/ActivityList.tsx]: Nice!! Good job figuring out how to define an interface, functional component AND a list of Activities with typescript! 👏

File: src/App.tsx
Comment (angelkbrown) [inline src/App.tsx]: Don't be afraid to delete! That's what's good about version control -- we can always look at past versions if we need to add something back in.

**Review by angelkbrown — APPROVED** [review]
Nice work, Victoria! 🎉 I'll defer to Darga for any more advice on implementation details, especially around the typescript options. But I ran your branch locally and it worked great for me! 👍

File: src/ActivityList.tsx
Comment (dargaCode) [reply src/ActivityList.tsx]: Yes well done!

File: src/main.tsx
Comment (dargaCode) [inline src/main.tsx]: You should only have to add these comments to your own new files - I think it should be safe to revert this file?

File: src/ActivityList.tsx
Comment (dargaCode) [inline src/ActivityList.tsx]: Eventually we're going to need this component to accept props. Can you please move this activities array out to App and then pass it into `ActivitiesList` as a prop called `activities`?

(In order to still give this array your custom type, you'll have to export `Activity` from here as a non-default export, and then import into `App` along with the `ActivitiesList`:

`import ActivitiesList, { Activity } from "./components/ActivitiesList/ActivitiesList"`
)

File: src/ActivityList.tsx
Comment (dargaCode) [inline src/ActivityList.tsx]: yessss 💪

File: src/ActivityList.tsx
Comment (dargaCode) [inline src/ActivityList.tsx:1]: Please rename this component and file to `ActivitiesList` and move the file into a new folder called `src/components/ActivitiesList`

Sorry I forgot to mention this in the ticket!

File: src/App.tsx
Comment (dargaCode) [inline src/App.tsx]: For now please leave the old code in App as well, and add ActivitiesList to the render right below HelloWorld

(We can come back later on and remove these other components)

File: src/App.tsx
Comment (dargaCode) [inline src/App.tsx]: I know you said the import isn't working correctly. After you move the `ActivityList` to the new folder inside `/components`, please try following the same format as the examples on lines 2-3. 

We're using the `@` shortcuts to make it easier to import from important folders without ever having to use `../../../` to go up to parent folders. 

You can also see here that the import for this project expects the file extensions to be left off entirely:

```suggestion
import ActivityList, { Activity } from "@components/ActivityList/ActivityList";
```

**Review by dargaCode — CHANGES_REQUESTED** [review]
Great job Victoria! 

Sorry there were a couple of details here that I forgot to add to the ticket. Please make these changes and then when you've re-pushed, you can "re-request review" from me next to my name in the upper right of the page 🙏

**Review by dargaCode — APPROVED** [review]
Awesome thank you! Approved =D

Please don't merge yet though - In the team meeting we can discuss  blending together aspects of these 3 different PRs

**Pull request merged**
